DOCS-1201: Update `ultrasonic` sensor docs and document `ultrasonic` camera (#1939)
viamrobotics/docs: docs/components/camera/ultrasonic.md

---
title: "Configure an ultrasonic Camera"
linkTitle: "ultrasonic"
weight: 60
type: "docs"
description: "Configure an ultrasonic model camera."
tags: ["camera", "components", "ultrasonic"]
icon: "/icons/components/camera.svg"
images: ["/icons/components/camera.svg"]
# SME: #team-bucket
---

{{< alert title="Usage" color="tip" >}}

> An ultrasonic distance sensor can also be configured as a [sensor](/components/sensor/) resource.
> [Configure it as such](/components/sensor/ultrasonic/) to utilize the [sensor API](/components/sensor/#api) with your ultrasonic sensor hardware.

Reviewer: Removes some duplication and the "as such" which feels a bit roundabout
```suggestion
An ultrasonic distance sensor can also be configured as a [sensor](/components/sensor/) resource, if you need to use the [sensor API](/components/sensor/#api) with your ultrasonic sensor hardware.
```

> An ultrasonic distance sensor can also be configured as a [sensor](/components/sensor/) resource.
> [Configure it as such](/components/sensor/ultrasonic/) to utilize the [sensor API](/components/sensor/#api) with your ultrasonic sensor hardware.
> 
> Configure your ultrasonic sensor as a camera if you want to use it as an input to a [vision service](/services/vision/) model that returns obstacles.
> As a camera model, the ultrasonic model implements the camera method [`GetPointCloud()`](/components/camera/#getpointcloud), rather than the sensor method [`GetReadings()`](/components/sensor/#getreadings).

Reviewer: ```suggestion
An ultrasonic distance sensor can also be configured as a [sensor](/components/sensor/) resource.
When configured as a sensor, you can use the [sensor API](/components/sensor/#api):

- [`GetReadings()`](/components/sensor/#getreadings)
```

Author: Why not have the input to vision service obstacles part? @npentrel

Author: Actually, I see what you mean-- makes more sense with the suggestion below for sensor page also calling that out. done
